test/tools/obdscanner.uts

```diff
@@ -104,14 +102,16 @@ if p.wait() == 0:
 + Syntax check
 
 = Import isotp
-conf.contribs['ISOTP'] = {'use-can-isotp-kernel-module': False}
+conf.contribs['ISOTP'] = {'use-can-isotp-kernel-module': ISOTP_KERNEL_MODULE_AVAILABLE}
```

Comment:
Could you explain theses changes?

Reply:
I did not see a reason why this should be `False`. There is a variable that is `True`  if the kernel module exists. And in my opinion if it exists it should be used.